### ISSUE TITLE
The language server doesn't work for haxeflixel projects

### ISSUE THREAD

**Issue by gamemake-eng** [Frixuu/Zed-Haxe#4]
Whenever I open a haxeflixel project, I get errors saying "type not found, flixel.(insert class here)". I have haxeflixel installed and working. I dont know what is actually causing this.

**Comment by gamemake-eng** [comment]
Turns out vshaxe doesn't recognize Project.xml files. You would need a separate extension for that. I think people should probably know this.

**Comment by Jamextreme140** [comment]
Yeah, just like VSCode, for OpenFL/Lime project it requires the Lime extension.

**Comment by mobilex1122** [comment]
For anyone who needs suggestions for lime project there is a workaround that uses the compilation `.hxml` files located in the export dirs (`<output>/<target>/haxe/debug.hxml`).

So you can do something like:
```json
{
  "lsp": {
    "haxe-language-server": {
      "initialization_options": {
        "displayArguments": ["Export/html5/haxe/debug.hxml"]
      }
    }
  }
}
```

Source: https://github.com/vshaxe/vshaxe/issues/606#issuecomment-1733834758

But a dedicated Lime Extension would be the best solution to this. (Could also work as a Debug Adapter)
